prediction/src/algorithms/segment/src/training.py

```diff
@@ -0,0 +1,87 @@
+import glob
+import math
+import os
+
+import numpy as np
+import pylidc as pl
+from keras.callbacks import ModelCheckpoint
+from keras.models import load_model
+from src.preprocess.lung_segmentation import save_lung_segments
+
+from .data_generation import get_dicom_paths
+from .model import simple_model_3d
+
+
+def get_data_shape():
+    dicom_paths = get_dicom_paths()
+    max_x, max_y, max_z = get_max_scaled_dimensions(dicom_paths)
+    return 128, 128, 128, 1  # TODO: dynamically crop shape based on maximal axes
```

Comment:
We need a fixed input (and thus output) size of the model. Something like taking the biggest spreads in every dimension would be possible (as it would be implemented by `get_max_scaled_dimensions`) but we would then be supposed to pad all images to this size. My memory was not enough for this so I decided to temporarily set the shape to a smaller, fixed one.